x-pack/plugins/session_view/server/routes/alerts_route.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+import type { ElasticsearchClient } from 'kibana/server';
+import { IRouter } from '../../../../../src/core/server';
+import {
+  ALERTS_ROUTE,
+  ALERTS_PER_PAGE,
+  ALERTS_INDEX,
+  ENTRY_SESSION_ENTITY_ID_PROPERTY,
+} from '../../common/constants';
+import { expandDottedObject } from '../../common/utils/expand_dotted_object';
+
+export const registerAlertsRoute = (router: IRouter) => {
+  router.get(
+    {
+      path: ALERTS_ROUTE,
```

Comment:
Examples of this would be appreciated, not sure how to go about accessing AlertsClient after making rule_registry a dep. I assume it's not as simple as just importing AlertsClient from rule_registry/server/index.ts

Reply:
@mitodrummer you can use the `rule_registry` plugin start contract `RuleRegistryPluginStartContract` for accessing AlertsClient, by calling the method `getRacClientWithRequest`. 
I didn't find the code for AlertsClient , but here is the similar usage of RulesClient: you get it by calling `getRulesClientWithRequest`, code x-pack/plugins/monitoring/server/plugin.ts

https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/server/plugin.ts#L333-L340

Reply:
rule_registry is added as a dep and I'm making use of getRacClientWithRequest in alerts_route.ts. Let me know if the plugin setup looks good.